Server/JokesOnYou.Web.Api/Extensions/ServiceExtensions.cs

```diff
@@ -18,8 +21,29 @@ public static IServiceCollection ConfigureAppServices(this IServiceCollection se
             services.AddDbContext<DataContext>(options =>
                 options.UseSqlite(config.GetConnectionString("DefaultConnection")));
 
-            services.AddIdentity<User, IdentityRole>().AddEntityFrameworkStores<DataContext>();
+            services.AddIdentity<User, IdentityRole>(
+                options => { options.SignIn.RequireConfirmedAccount = false; options.User.RequireUniqueEmail = true; }
```

Comment:
sign in requireconfirmed account is fale by default i believe. Might wanna check if same applies to requniqmail

Reply:
Property                         | Description                                           | Default
RequireConfirmedEmail | Requires a confirmed email to sign in. | false


Property                   | Description                                                    | Default
RequireUniqueEmail | Requires each user to have a unique email. | false